docs/en/core/defaults-files.md

```diff
@@ -79,3 +79,47 @@ variables:
 ```
 
 You can also make use of this templating system yourself. If you want to write a custom template, you can add statements such as `$if(myvariable)$$myvariable$$endif$` and insert the variable `myvariable` into any frontmatter or defaults file, and it will be replaced with whatever value you set it to.
+
+
+### Microsoft Word Template (.docx)
+
+For export as a Word document, you need a .docx document that serves as a template. This file should be stored in the same folder as your markdown files and added to the Word export Defaults file in Assets Manager. 
+
+#### Reference Document 
+
+Use an existing Word document or download the reference.docx from Pandoc and change the styles. Your reference doc is a Word file that you format to your preferences. You need to adjust the Word Styles (Paragraph, Heading, Body Text, Footnote Text, etc) for it to apply. 
```

Comment:
for the repository, I don't see the download directory option, I only have the option to download the whole pandoc repository as a .zip

<img width="750" alt="image" src="https://user-images.githubusercontent.com/109297886/200011258-56246ae7-bbc1-4ed8-8289-2a9ea0283e37.png">

Reply:
Ouh, alright, then it's probably the GitHub extension I have installed. Then … nevermind. Feel free to adapt the wording so that it's correct (zipping that directory and renaming it from `.zip` to `.docx` will still work)

Reply:
I tried to do that and it didn't work for me. I think it worked in the past though, so I assume it'll work for others. I included it in the updated file anyways.

Reply:
This is more of a request, but maybe you could include a copy of the reference documents in the Zettlr or Zettlr docs repos, and then add a link to it in documentation, so that it's easier for people to find and download. 

I don't really have much experience with command lines or github, so personally, it was a complicated and time consuming experience in trying to find the reference document to edit. I couldn't figure out where Pandoc was saving the doc or what I needed to change for it to show up or how to download it from github.